catalog/catalog-app/src/main/resources/features.xml

```diff
@@ -39,15 +39,15 @@
 
     <feature name="catalog-core" install="manual" version="${project.version}"
              description="Catalog Core feature containing the API, third party bundles necessary to run ddf-core.">
-        <feature>catalog-core-api</feature>
+        <feature prerequisite="true">catalog-core-api</feature>
```

Comment:
I couldn't find a good explanation of the `prerequesite` attribute anywhere but I assume that in this case, the bundles inside the `catalog-core` feature won't be started until all of the ones in `catalog-code-api` are up. Does this impact the startup time? If so, do we really need this restriction?